Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -567,6 +567,10 @@ theorem trans {a o o' : Ordinal.{u}} {f : ∀ b < o, Ordinal.{u}} (hf : IsFundam
     · exact hf.2.2
     · exact hg.2.2
 
+theorem lt {a o : Ordinal} {s : Π p < o, Ordinal}
```

Comment:
I think this lemma is fine given the current API. That said, it'd be nice to at some point redefine fundamental sequences as functions `Iio o → Ordinal` (and probably, move all of this to its own file `Ordinal/FundamentalSequence` too).

Reply:
I was thinking the same thing, once `iSup` is used everywhere.

Reply:
I think that this lemma should be `protected`, otherwise I trust @vihdzp on this matter.
bors d+
```suggestion
protected theorem lt {a o : Ordinal} {s : Π p < o, Ordinal}
```